webhooks/grading/grading.js

```diff
@@ -30,8 +34,24 @@ function processResults(data) {
 router.post('/', function(req, res, next) {
 
     const data = req.body;
+    if (data.event === 'grading_started') {
+        if (data.job_id === undefined || data.job_id === null || !Number.isInteger(data.job_id)) {
```

Comment:
A slightly evil but useful JS idiom is to use `job_id == null` for this, which is true if and only if `job_id` is either `undefined` or `null`. I do use this myself elsewhere, but I'm not sure whether it's elegant or horrific. In this particular case, however, I think it's enough to just replace the whole line with `if (!Number.isInteger())` which will correctly handle the undefined and null cases.